### ISSUE TITLE
scheme doesn't work any more

### ISSUE THREAD

**Issue by linas** [opencog/opencog#2020]
The following gives an error:
```
$ guile
scheme@(guile-user)> (use-modules (opencog) (opencog atom-types))
While compiling expression:
ERROR: In procedure primitive-load-path: Unable to find file "opencog/nlp/types/nlp_types.scm" in load path
scheme@(guile-user)> 
```

**Comment by linas** [comment]
This commit broke it:
https://github.com/opencog/opencog/commit/614b137c14ab72bc5ff1f22b32eb7b82b1a2f8a6

How do you test?

**Comment by linas** [comment]
This diff fixes it:
```
diff --git a/opencog/scm/atom-types.scm b/opencog/scm/atom-types.scm
index 9e2186a..13fdab2 100644
--- a/opencog/scm/atom-types.scm
+++ b/opencog/scm/atom-types.scm
@@ -25,7 +25,7 @@
 
 (add-to-load-path "/usr/local/share/opencog/scm")
 (use-modules (opencog))  ; needed for cog-type->int
-(load-from-path "opencog/nlp/types/nlp_types.scm")
-(load-from-path "opencog/spacetime/spacetime_types.scm")
-(load-from-path "opencog/attention/attention_types.scm")
-(load-from-path "opencog/embodiment/embodiment_types.scm")
+(load-from-path "nlp/types/nlp_types.scm")
+(load-from-path "spacetime/spacetime_types.scm")
+(load-from-path "attention/attention_types.scm")
+(load-from-path "embodiment/embodiment_types.scm")
```

**Comment by AmeBel** [comment]
I run `./opencog/cogserver/server/cogserver -c ../lib/development.conf ` from the build directory. And

```
opencog@e30aa684da9b:/opencog/build$ rlwrap telnet localhost 17001
Trying ::1...
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
opencog> (use-modules (opencog atom-types))
Entering scheme shell; use ^D or a single . on a line by itself to exit.
guile> 
guile> (WordNode "what")
(WordNode "what")
```

Sorry for the inconveniance. From here on, I won't touch any of your scheme code or merge any that touches them without your review.

**Comment by linas** [comment]
Ah.  That would explain a lot. ... I never run from the build directory, and I never start the cogserver from the shell; I start it from the guile prompt, as `(start-cogserver "/some/cogserver.conf")`  So I kind of expect things to work, no matter what directory you might happen to be in at that moment, and no matter how the cogserver is started.   The current working directory should not affect operation.

**Comment by linas** [comment]
Fixed merged in pull req #2025, so closing.
